bundles/EcommerceFrameworkBundle/FilterService/FilterType/ElasticSearch/MultiSelectFromMultiSelect.php

```diff
@@ -23,10 +23,13 @@ class MultiSelectFromMultiSelect extends \Pimcore\Bundle\EcommerceFrameworkBundl
 {
     public function prepareGroupByValues(AbstractFilterDefinitionType $filterDefinition, ProductListInterface $productList)
     {
-        if ($filterDefinition instanceof FilterMultiSelectFromMultiSelect) {
-            $field = $this->getField($filterDefinition);
-            $productList->prepareGroupByValues($field, true, !$filterDefinition->getUseAndCondition());
+        $useAndCondition = false;
```

Comment:
why change that? 

we should check `$filterDefinition instanceof \Pimcore\Model\DataObject\Fieldcollection\Data\FilterMultiSelectFromMultiSelect`, other wise throw `InvalidConfigException`